### ISSUE TITLE
Taglib refuses specific 24bit/48kHz Flac files

### ISSUE THREAD

**Issue by ghlupe2** [taglib/taglib#985]
The Gerbera people sent me here to report this issue so I compiled the tagreader example code and could verify the issue:

I've got quite some 24bit/48kHz Flac files that taglib refuses to process (prints only ****path/to/file.flac****). Foobar2000 (and VLC as well btw.) reads the metadata okay and after re-sampling a test file with foobar2000 flac converter it gets processed by taglib as well. 

The original files (from B&W Society of Sound) are quite old. Hex reader shows "reference libFLAC 1.1.3 20070917" while the re-sampled file shows "reference libFLAC 1.3.2 20170101". Is there a known limitation regarding early hi-res flac files?

**Comment by ufleisch** [comment]
Could you please provide a link to some example file?

**Comment by ghlupe2** [comment]
Sure: testfile

**Comment by ghlupe2** [comment]
[testfile](https://drive.google.com/file/d/1cVTYL2sswKcJtzEnKy7STuXiGv_eWpF8/view?usp=sharing)

Sorry, didn't work on my mobile

**Comment by sbooth** [comment]
I'm not able to reproduce this using the current head (1721354):

```
% git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
% ./tagreader ~/Downloads/24-48test.flac
******************** "/*/24-48test.flac" ********************
-- TAG (basic) --
title   - "Piano Concerto no. 1: III. Moderato"
artist  - "Dmitri Dmitrijewitsch Schostakowitsch"
album   - "Shostakovich: Piano Concertos nos. 1 & 2 / Shchedrin: Piano Concerto no. 5"
year    - "2012"
comment - ""
track   - "3"
genre   - ""
-- TAG (properties) --
ALBUM                      - "Shostakovich: Piano Concertos nos. 1 & 2 / Shchedrin: Piano Concerto no. 5"
ALBUMARTIST                - "Dmitri Dmitrijewitsch Schostakowitsch, Rodion Konstantinovich Shchedrin; Denis Matsuev, Mariinsky Theatre Orchestra, Valery Abisalovich Gergiev"
ALBUMARTISTSORT            - "Schostakowitsch, Dmitri Dmitrijewitsch, Shchedrin, Rodion Konstantinovich; Matsuev, Denis, Mariinsky Theatre Orchestra, Gergiev, Valery Abisalovich"
ARTIST                     - "Dmitri Dmitrijewitsch Schostakowitsch"
ARTISTS                    - "Dmitri Dmitrijewitsch Schostakowitsch"
ARTISTSORT                 - "Schostakowitsch, Dmitri Dmitrijewitsch"
CATALOGNUMBER              - "MAR0509"
COMPOSER                   - "Dmitri Dmitrievich Shostakovich"
COMPOSERSORT               - "Shostakovich, Dmitri Dmitrievich"
CONDUCTOR                  - "Valery Abisalovich Gergiev"
DATE                       - "2012"
DISCNUMBER                 - "1"
DISCTOTAL                  - "1"
LABEL                      - "Mariinsky"
LANGUAGE                   - "zxx"
MEDIA                      - "Digital Media"
MT_FORMAT                  - "FLAC 24/48 stereo"
MUSICBRAINZ_ALBUMARTISTID  - "519dd32e-8f30-4380-8826-7aa99169e1bb"
MUSICBRAINZ_ALBUMARTISTID  - "09deb4a5-8746-47f8-b020-cde50023841d"
MUSICBRAINZ_ALBUMARTISTID  - "981e0034-de98-4680-9822-9fc3eda40f04"
MUSICBRAINZ_ALBUMARTISTID  - "6c05e40f-33b2-4150-b93a-ddc2b0d88a6a"
MUSICBRAINZ_ALBUMARTISTID  - "87b4252d-314b-4252-9cd3-44c08f3087b6"
MUSICBRAINZ_ALBUMID        - "f9432efc-22ea-461d-9b6c-8e5bb5f892c1"
MUSICBRAINZ_ARTISTID       - "519dd32e-8f30-4380-8826-7aa99169e1bb"
MUSICBRAINZ_RELEASEGROUPID - "d3a18dd4-c0a4-4787-b95d-0bb1f65825f8"
MUSICBRAINZ_RELEASETRACKID - "491d45f6-51ae-44c7-8355-d694de47b39c"
MUSICBRAINZ_TRACKID        - "d92f70ec-4aa7-4eb1-ba30-aee4449fbdd2"
MUSICBRAINZ_WORKID         - "15a890d3-40d0-3943-9b4d-6cb4037f7c34"
ORIGINALDATE               - "2012"
ORIGINALYEAR               - "2012"
PERFORMER                  - "Mariinsky Theatre Orchestra (orchestra)"
PERFORMER                  - "Denis Matsuev (piano)"
RELEASECOUNTRY             - "XW"
RELEASESTATUS              - "official"
RELEASETYPE                - "album"
SCRIPT                     - "Latn"
TITLE                      - "Piano Concerto no. 1: III. Moderato"
TRACKNUMBER                - "3"
TRACKTOTAL                 - "10"
WORK                       - "Concerto for Piano, Trumpet and String Orchestra no. 1 in C minor, op. 35: III. Moderato"
-- AUDIO --
bitrate     - 1276
sample rate - 48000
channels    - 2
length      - 1:19
```

What version of taglib are you using?

**Comment by ghlupe2** [comment]
taglib-config says 1.11.1

**Comment by ufleisch** [comment]
I see all the tags when using the TagLib master branch, but no tags with 1.11.1, so this was probably fixed in one of the commits since version 1.11.1.

**Comment by ghlupe2** [comment]
Thanks a lot, I am going to try to run Gerbera with the current head.
